### PR TITLE
Jekyll update

### PR DESCRIPTION
**Pull request by JRPearson500**
Migration to expanded site.  No update to projects - coming

### CODE REVIEW

File: _data/completed.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
+- title: P34 - Using Hypergraphs to Investigate the Impact of Comorbidities  
+  codelink: "https://github.com/nhsx/hypergraph-testing"
+  reportlink: ComingAutumn2023
```
Comment (danjscho) [inline _data/completed.yml]: If not a url seems to hit a 404, have we a better way of handling this whilst there isn't a public facing aspect of either the code or the report?

File: _data/data-OSOCpost/sc03_fig01_nattrends_REWORKED.png
Comment (danjscho) [inline _data/data-OSOCpost/sc03_fig01_nattrends_REWORKED.png:1]: To remove

File: _data/data-OSOCpost/sc03_fig05_pracnatcoverage_REWORKED.png
Comment (danjscho) [inline _data/data-OSOCpost/sc03_fig05_pracnatcoverage_REWORKED.png:1]: To remove

File: _layouts/completed.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  {% for proj in site.data.completed %}{% if stage.name == proj.stage %}
+
+  <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
+    <div class="nhsuk-card nhsuk-card--clickable">
+      <div class="nhsuk-card__content">
+        <h2 class="nhsuk-card__heading nhsuk-heading-m">
+          <a class="nhsuk-card__link" href="{{proj.codelink}}">{{proj.title}}</a>
+        </h2>
+        <h6 class="post-author" itemprop="author">{{proj.author}}</h6>
+        <p class="nhsuk-card__description"> {{proj.description}} <a href={{proj.reportlink}}>Published Report</a>  </p>
```
Comment (danjscho) [inline _layouts/completed.html]: Suggest giving a minimum of a new line between description and report link.

File: content/feedback.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+permalink: feedback.html
+---
+
+# {{page.title}}
+
+<div class="nhsuk-expander-group">
+  <details class="nhsuk-details nhsuk-expander">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text">
+        David Brind - June 2022
```
Comment (danjscho) [inline content/feedback.md]: Worth adding the wave they were in for consistency across the pages.

File: content/projects.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+|![area](https://img.shields.io/badge/NLP-yellow) &nbsp; ![technique](https://img.shields.io/badge/STM-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TextData-9cf)|[Structural Topic Models and Sentiment Model Robustness - Next Steps](https://nhsx.github.io/nhsx-internship-projects/stm-sentiment-robustness/)|
+|![area](https://img.shields.io/badge/Synthetic-brightgreen) &nbsp; ![technique](https://img.shields.io/badge/ShadowModels-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[Synthetic Adversarial Suite - Next Steps](https://nhsx.github.io/nhsx-internship-projects/synthetic-adversarial-suite/)| 
+
+
+---
+## Closed Projects
+*Completed Projects are ones which our previous interns have finished and reported on.  There may still be many areas or investigation possible with these and so happy to talk about redeveloping these.
```
Comment (danjscho) [inline content/projects.md]: Missing closing `*` for italics

File: content/projects.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
+layout: base
+title: Project Proposals
+description: list of projects
+permalink: projects.html
+---
+
+# {{page.title}}
+
+Below is a list of internship projects which are continually being updated and added to. We also welcome project proposals from both prospective students and interested organisations.  Please contact us at our [mailbox](england.tdau@nhs.net) if you would have queries or would like to discus a proposal.
```
Comment (danjscho) [inline content/projects.md]: Is this the right mailbox for now?

Comment (danjscho) [reply content/projects.md]: `discus` -> `discuss`

File: content/projects/active/active.md
Comment (danjscho) [inline content/projects/active/active.md:1]: To be archived?

File: content/projects/archive/archive.md
Comment (danjscho) [inline content/projects/archive/archive.md:1]: To be archived?

File: content/projects/new_open/open.md
Comment (danjscho) [inline content/projects/new_open/open.md:1]: To be archived?

File: content/waysofworking.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+## Project Structure
+
+As these are short-term and remote projects we propose a minimum of weekly project *"check-ins"* with the NHS project supervisor (although more than often the frequency is higher than this).  Regular code reviews & pair programming are encouraged as this increases the quality of code as well as ensuring the NHS supervisor understands the detailed progress of the work.  These meetings will focus on technical specifics and hurdles.
+ 
+More formal milestone meetings are also requested in order to run the project past the innovation team for wider comment.  These meetings will focus on deliverables and project timeline and are usually monthly presented to a wider audience who are interested in the work.  
+ 
+It is expected at the beginning of the project that these milestones will be agreed between candidate and supervisor, and put in place to support monitoring of the project timelines, but should reflect releases of product / stages of the project where possible. 
+ 
+<p align="center"> 
+  <img src="assets/img/milestoneTimeline.png" alt="Rough project structure with monthly milestones and linear sprints" width="800"/>
```
Comment (danjscho) [inline content/waysofworking.md]: Suggest width `600` as image is slightly pixelated at `800`

File: index.md [hunk 7]
```diff
@@ -1,144 +1,51 @@
...
-- ![area](https://img.shields.io/badge/NLP-yellow)
-![technique](https://img.shields.io/badge/SematicSearch-lightgrey)
-![datamodality](https://img.shields.io/badge/MultiModalData-blueviolet) &nbsp; [NHS Semantic Search](https://nhsx.github.io/nhsx-internship-projects/nhs-semantic-search/)
-
----
-More information on the internship scheme can be found on the [NHS England internship scheme for innovation and analytics in health](https://www.nhsx.nhs.uk/key-tools-and-info/nhsx-analytics-unit/nhsx-internship-scheme-innovation-and-analytics-health/) website.
+# {{page.title}}
+
+These internships are aimed at current PhD students looking for an industrial placement of around 5-months with the right to work in the UK.  The projects are focussed on innovation in particular around getting the most value out of NHS data.  They often have a focus on emerging data science techniques and so we advertise mainly to data science programmes however previous interns have included disciplines such as clinical, mathematics, computer science and bioinformatics which have added huge value through the range of approaches and knowledge.
```
Comment (danjscho) [inline index.md]: `The projects are focussed on innovation in particular around getting the most value out of NHS data.` -> `The projects are focussed on innovation, in particular around getting the most value out of NHS data.`

Comment (danjscho) [reply index.md]: `however previous interns have included disciplines such as clinical,` -> `however previous interns have come from other disciplines such as clinical,`

File: index.md [hunk 8]
```diff
@@ -1,144 +1,51 @@
...
+
+For details around **how to apply** see the "Scheme Overview" page.  For details around **open projects** see the "Projects" page.
```
Comment (danjscho) [inline index.md]: Make the `Scheme Overview` and `Projects` references into links to those pages.

File: index.md [hunk 9]
```diff
@@ -1,144 +1,51 @@
...
+### Current Projects 
+Currently our interns are working on the following six projects in two waves.  These are the original briefs they applied to.  Their work and outputs will be available on our organisation github.
+
+|:---:|--|
+|January 2023|
```
Comment (danjscho) [inline index.md]: Add Wave number to standardise across pages

File: index.md [hunk 10]
```diff
@@ -1,144 +1,51 @@
...
+|![area](https://img.shields.io/badge/NLP-yellow) &nbsp; ![technique](https://img.shields.io/badge/CogStack-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TextData-9cf) &emsp; &emsp; &emsp; &emsp; &emsp; |[Enriching Neurology Patient Information using MedCAT](https://nhsx.github.io/nhsx-internship-projects/enriching-neurology-information-medcat/)|
+|![area](https://img.shields.io/badge/Synthetic-brightgreen) &nbsp; ![technique](https://img.shields.io/badge/VAE-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[SynthVAE - Next Steps](https://nhsx.github.io/nhsx-internship-projects/synthvae-extension/)|
+|![area](https://img.shields.io/badge/Graphs-yellowgreen) &nbsp; ![technique](https://img.shields.io/badge/Hypergraphs-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[Transforming Healthcare Data with Graph-based Techniques - Next Steps](https://nhsx.github.io/nhsx-internship-projects/transforming-healthcare-data-graph-based-sail-update/)|
+|June 2023|
```
Comment (danjscho) [inline index.md]: Add Wave number to standardise across pages

File: index.md [hunk 11]
```diff
@@ -1,144 +1,51 @@
...
+
+<ul class="nhsuk-grid-row nhsuk-card-group">
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+    <div class="nhsuk-card nhsuk-card--clickable">
+        <img class="nhsuk-card__img" src="assets/img/graph.jpg" alt="teaser" />
+        <div class="nhsuk-card__content">
+            <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="projects.html">Projects</a>
+            </h2>
+            <p class="nhsuk-card__description">List of active and open projects to either get involved with or apply to in our next round</p>
```
Comment (danjscho) [inline index.md:34]: Can we update the image widths/heights(?) to make the text appear at the same point across the two panels?

File: index.md [hunk 12]
```diff
@@ -1,144 +1,51 @@
...
+            <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="overview.html">Scheme Overview</a>
+            </h2>
+            <p class="nhsuk-card__description">Details of aims and application process of the scheme</p>
+        </div>
+    </div>
+  </li>
+</ul>
+
+Please get in contact by emailing **england.tdau@nhs.net** with further questions around the internship.
```
Comment (danjscho) [inline index.md]: Needs updating to be hyperlink (currently just bold). Also confirm this is the correct email address at the moment.

**Review by danjscho — CHANGES_REQUESTED** [review]
Added comments - nothing major to change, just some suggestions/typos/minor updates.

File: _data/data-OSOCpost/sc03_fig01_nattrends_REWORKED.png
Comment (JRPearson500) [reply _data/data-OSOCpost/sc03_fig01_nattrends_REWORKED.png:1]: removed

File: _data/data-OSOCpost/sc03_fig05_pracnatcoverage_REWORKED.png
Comment (JRPearson500) [reply _data/data-OSOCpost/sc03_fig05_pracnatcoverage_REWORKED.png:1]: removed

File: _data/completed.yml (hunk 1 above)
Comment (JRPearson500) [reply _data/completed.yml]: consider holder page

File: content/feedback.md (hunk 3 above)
Comment (JRPearson500) [reply content/feedback.md]: Add to Home and feedback

File: content/projects.md (hunk 5 above)
Comment (JRPearson500) [reply content/projects.md]: find and replace to datascience@nhs.net

File: content/projects/archive/archive.md
Comment (JRPearson500) [reply content/projects/archive/archive.md:1]: investigate collaspable

File: index.md (hunk 12 above)
Comment (JRPearson500) [reply index.md]: add form of mailto:

File: _data/completed.yml (hunk 1 above)
Comment (JRPearson500) [reply _data/completed.yml]: Implemented Holding Page

File: _layouts/completed.html (hunk 2 above)
Comment (JRPearson500) [reply _layouts/completed.html]: Added in <p></p> around Published report

File: content/feedback.md (hunk 3 above)
Comment (JRPearson500) [reply content/feedback.md]: Implemented

File: content/projects.md (hunk 5 above)
Comment (JRPearson500) [reply content/projects.md]: Three corrections to <a href="mailto:datascience@nhs.net">our mailbox</a>

File: content/projects/new_open/open.md
Comment (JRPearson500) [reply content/projects/new_open/open.md:1]: Going to keep the folders as useful sorting for the project briefs

File: content/projects/archive/archive.md
Comment (JRPearson500) [reply content/projects/archive/archive.md:1]: Going to keep the folders as useful sorting for the project briefs

Comment (JRPearson500) [reply content/projects/archive/archive.md:1]: Tried collapsible but it can't cope with the table formatting

File: content/projects/active/active.md
Comment (JRPearson500) [reply content/projects/active/active.md:1]: going to leave the folder as usal organisaing

File: content/waysofworking.md (hunk 6 above)
Comment (JRPearson500) [reply content/waysofworking.md]: Implemented

File: index.md (hunk 8 above)
Comment (JRPearson500) [reply index.md]: Implemented

File: index.md (hunk 9 above)
Comment (JRPearson500) [reply index.md]: Done

File: index.md (hunk 10 above)
Comment (JRPearson500) [reply index.md]: Done

File: index.md (hunk 11 above)
Comment (JRPearson500) [reply index.md:34]: height = "250"

File: index.md (hunk 12 above)
Comment (JRPearson500) [reply index.md]: added

**Review by danjscho — APPROVED** [review]

**Pull request merged**